src/app/components/PageLayoutWrapper/index.tsx

```diff
@@ -74,23 +76,7 @@ const PageLayoutWrapper = ({
 
     const topStories = pageData.secondaryColumn?.topStories;
     const mostReadItems = pageData.mostRead?.items;
-    const servicesForVariantA = ['pidgin', 'hindi', 'urdu', 'uzbek', 'hausa'];
-    const servicesForVariantB = ['mundo', 'burmese', 'arabic'];
 
-    let variantValue;
-
-    if (servicesForVariantA.includes(service)) {
-      variantValue = 'A';
-    } else if (servicesForVariantB.includes(service)) {
-      variantValue = 'B';
-    } else {
-      variantValue = 'none';
-    }
-    const experimentVariant: 'A' | 'B' | 'none' = ['A', 'B'].includes(
-      variantValue,
-    )
-      ? (variantValue as 'A' | 'B')
-      : 'none';
     let dataForOJExperiment;
     if (experimentVariant === 'A') {
```

Comment:
The variation names will need to match those defined in the Optimizely experiment.